lib/lookup.json

```
@@ -262,11 +262,11 @@
     "prefix": "v",
     "maintainers": ["cpojer", "scotthovestadt", "SimenB", "thymikee", "jeysal"],
     "yarn": true,
+    "install": ["install", "--no-immutable"],
     "scripts": ["remove-examples", "build:js", "test-ci-partial"],
```

```suggestion
    "scripts": ["build:js", "remove-examples", "test-ci-partial"],
```

hopefully this should work 🙂